modules/packages/conan/reference.go

```diff
@@ -56,7 +54,7 @@ func NewRecipeReference(name, version, user, channel, revision string) (*RecipeR
 	if !namePattern.MatchString(name) {
 		return nil, ErrValidation
 	}
-	if _, err := goversion.NewSemver(version); err != nil {
+	if version == "" {
```

Comment:
```suggestion
	if strings.TrimSpace(version) == "" {
```
I would disallow whitespaces.